servicecontrol/servicecontrol-in-practice.md

```diff
@@ -0,0 +1,28 @@
+---
+title: ServiceControl in Practice
+summary: Details how to think about the use of ServiceControl in production environments
+tags:
+- ServiceControl
+---
+
+ServiceControl is a tool that can accommodate many situations. Features can be enabled or disabled depending on the needs of the environment. Each feature is valuable in certain contexts but also comes with a cost in terms of resource usage and performance.
+
+For example, in a development environment the [Debug Session](/servicecontrol/plugins/debug-session.md) and [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugins provide detailed information to support problem analysis. Particular advise that customers not use these  plugin outside of a development environment.
```

Comment:
@mikeminutillo because this :)

> DANGER: For Development only. This plugin will result in a significant increase in the load placed on ServiceControl. As such it should not be used in production.